frappe/oauth.py

```diff
@@ -205,7 +207,7 @@ def save_bearer_token(self, token, request, *args, **kwargs):
 		otoken.user = request.user
 		otoken.scopes = get_url_delimiter().join(request.scopes)
 		otoken.access_token = token['access_token']
-		otoken.refresh_token = token['refresh_token']
+		otoken.refresh_token = token.get('refresh_token')
```

Comment:
There is no refresh_token in case of Implicit Grant (Useful for Mobile/JS Client Apps)
e.g.
```
http://0.0.0.0:8000/redir#access_token=6InqQCERxlsVXTzPjfdESxlQkNdtbh&token_type=Bearer&expires_in=3600&scope=all+openid
```